languages/common-lisp/exercises/concept/README.md

```diff
@@ -1,7 +1 @@
-
-
-
     This space left intentionally blank
-    
```

Comment:
@ErikSchierboom ^^

Reply:
Yeah. This is just a stub file. The actual white space doesn't matter I think. CC @exercism/common-lisp